autoload/which_key/window.vim

```diff
@@ -63,10 +63,10 @@ function! s:show_popup(rows) abort
   let offset = s:floating_win_col_offset()
   if g:which_key_floating_relative_win
     let col = offset + win_screenpos(g:which_key_origin_winid)[1]
-    let maxwidth = winwidth(g:which_key_origin_winid) - offset - 1
+    let maxwidth = winwidth(g:which_key_origin_winid) - offset + 1
   else
     let col = offset
-    let maxwidth = &columns - offset - 1
+    let maxwidth = &columns - offset + 1
```

Comment:
let maxwidth = &columns - offset

Reply:
> ```
>  let maxwidth = &columns - offset
> ```

It looks like Vim's popup uses column number starting from 1. Maybe we should use:
```
function! s:floating_win_col_offset() abort
  return (&number ? strlen(line('$')) : 0) + (&signcolumn ==# 'yes' ? 2: 0)
endfunction
...
let col = offset + 1
...
let maxwidth = winwidth(g:which_key_origin_winid) - offset
...
let maxwidth = &columns - offset
```
?